### ISSUE TITLE
Investigate botattr flag error checking

### ISSUE THREAD

**Issue by vanosg** [eggheads/eggdrop#464]
```
[19:12:48] #-HQ# botattr ztestbot +z
Bot flags for ztestbot are now +4.
```

**Comment by ghost** [comment]
May not ve valid any more, @vanosg 

\<Ami-Laptop> .botattr samantha +z
\<Agnes> Bot flags for samantha are now +iz.
\<Ami-Laptop> ./w samantha
\<SNIP>
\<Agnes>   BOT FLAGS: z

Can't duplicate.

Version: 1.8.3 / 1080303 (RC?) -- This is the current version via WGET GetEggdrop.com

**Comment by ghost** [comment]
Sorry; the test case should have been eggdrop -nt (HQ).

.botattr samantha -z
There are no bot flags for samantha.
.botattr samantha +z
Bot flags for samantha are now +z.
./w samantha
HANDLE                           PASS NOTES FLAGS           LAST
Samantha                         yes      0 bf              2017-07-21 (#BotHou
se)
\<SNIP>
  BOT FLAGS: z

**Comment by Cizzle** [comment]
This was fixed by commit dccd3fdb7847df3df124fd4ece11fb14f7053ad9.
